pom.xml

```diff
@@ -35,19 +35,32 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-
+		<!--
+		integration tests run by default when running the command "mvn verify"; can be overridden using the option
+		"-DskipITs=true"
+		-->
+		<skipITs>false</skipITs>
 		<junit.version>5.5.2</junit.version>
 	</properties>
 
 	<dependencies>
 		<!-- Main dependencies -->
-
+		<dependency>
+			<groupId>com.beust</groupId>
+			<artifactId>jcommander</artifactId>
+			<version>1.78</version>
+		</dependency>
 		<!-- required for javax.annotation.Nonnull -->
 		<dependency>
```

Comment:
I think the scope of this should be "test", as that's the only place it's used.

Reply:
Actually, I'm not even sure this is used anywhere. Maybe it can be removed?

Reply:
Sure, I thought IntelliJ had recommended it but it seems okay without it.

The usages and dependency have been removed: https://github.com/reactome/release-data-exporter/pull/4/commits/8fa20bfa30b96a0aa04cebad240cca56acb41910

Reply:
IntelliJ may think it's a great dependency to have, but if it's not getting _used_, then it's just an extra, unnecessary dependency.